cosmos-gui/conf/cosmos-gui.json

```diff
@@ -2,15 +2,27 @@
   "gui": {
     "port": 80
   },
+  "clusters": {
+    "storage": {
```

Comment:
This will be addressed in the following techdebt issue: https://github.com/telefonicaid/fiware-cosmos/issues/30